private/buf/cmd/buf/command/config/internal/internal.go

```diff
@@ -254,12 +253,16 @@ func lsRun(
 		}
 	} else {
 		allRulesOptions := []bufcheck.AllRulesOption{
-			bufcheck.WithPluginConfigs(bufYAMLFile.PluginConfigs()...),
+			bufcheck.WithPluginConfigs(workspace.PluginConfigs()...),
+		}
+		fileVersion := bufconfig.FileVersionV1
+		if workspace.IsV2() {
```

Comment:
We would need to return the `bufconfig.FileVersion` here, as there is currently no way to tell if the version is v1, v1beta1 or v2.

Reply:
Hmm, that doesn't make sense at a `Workspace` level though, since `v1` and `v1beta1` `buf.yaml` versions are module-level concerns... we may need to rethink this refactor a little bit.